NEWS

```diff
@@ -2,6 +2,23 @@ systemd System and Service Manager
 
 CHANGES WITH 243 in spe:
 
+        * Previously, filters defined with SystemCallFilter= would have the
+          effect that an calling an offending system call would terminate the
+          calling thread. This behaviour never made much sense, since killing
+          individual threads of unexpecting processes is likely to create more
+          problems than it solves. With this release the default action changed
+          from killing the thread to killing the whole process, which should be
```

Comment:
", which should be less problematic" → just drop this part. (Killing the whole process is not problematic at all, it just does the job.)